lib/core/src/lib/datatable/components/date-cell/date-cell.component.ts

```diff
@@ -15,67 +15,55 @@
  * limitations under the License.
  */
 
-import { Component, Optional, ViewEncapsulation } from '@angular/core';
+import { ChangeDetectionStrategy, Component, Input, OnInit, ViewEncapsulation } from '@angular/core';
 import { DataTableCellComponent } from '../datatable-cell/datatable-cell.component';
-import {
-    UserPreferencesService,
-    UserPreferenceValues
-} from '../../../common/services/user-preferences.service';
 import { AppConfigService } from '../../../app-config/app-config.service';
-import { takeUntil } from 'rxjs/operators';
-import { DataTableService } from '../../services/datatable.service';
+import { DateConfig } from '../../data/data-column.model';
+import { CommonModule } from '@angular/common';
+import { LocalizedDatePipe, TimeAgoPipe } from '../../../pipes';
 
 @Component({
+    standalone: true,
+    imports: [CommonModule, LocalizedDatePipe, TimeAgoPipe],
     selector: 'adf-date-cell',
-    template: `
-        <ng-container>
-            <span
-                title="{{ tooltip | adfLocalizedDate: 'medium' }}"
-                class="adf-datatable-cell-value"
-                *ngIf="format === 'timeAgo'; else standard_date">
-                {{ value$ | async | adfTimeAgo: currentLocale }}
-            </span>
-        </ng-container>
-        <ng-template #standard_date>
-            <span
-                class="adf-datatable-cell-value"
-                title="{{ tooltip | adfLocalizedDate: tooltipDateFormat }}">
-                {{ value$ | async | adfLocalizedDate: format }}
-            </span>
-        </ng-template>
-    `,
+    templateUrl: './date-cell.component.html',
     encapsulation: ViewEncapsulation.None,
-    host: { class: 'adf-date-cell adf-datatable-content-cell' }
+    host: { class: 'adf-date-cell adf-datatable-content-cell' },
```

Comment:
What about 'adf-date-cell' class, in other cases we just left 'adf-datatable-content-cell' only? We should leave it in this case?